Marlin/pins_GT2560.h

```diff
@@ -0,0 +1,99 @@
+
```

Comment:
I did a pin-by-pin comparison and found no differences. You can do the same, easily enough. Most pins defined as -1 should now be left out of pins files, so some of the pins defined as `-1` in `pins_GT2560.h` are not present in `pins_ULTIMAKER.h`.